src/IO/Observer/ObserverComponent.hpp

```diff
@@ -0,0 +1,43 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "AlgorithmNodegroup.hpp"
+#include "IO/Observer/ArrayComponentId.hpp"
+#include "IO/Observer/Initialize.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Parallel/Invoke.hpp"
+
+namespace observers {
+/*!
+ * \ingroup ObserversGroup
+ * \brief The nodegroup parallel component that is responsible for reducing data
+ * to be observed and for writing data to disk.
```

Comment:
I think we should clarify the ambiguity in nomenclature that we discussed at a call a while ago. When I read `observer` in a GR context I think of a tretrad construction, or in other words a collection of test particles that can make local measurements along their worldlines (see e.g. https://en.wikipedia.org/wiki/Frame_fields_in_general_relativity). Is this applicable to where you are going with this PR, or should we perhaps more clearly distinguish between physical observers in the above sense and the technical aspect of collecting and writing data to disk?

Reply:
I was not at whatever call discussed an ambiguity in nomenclature.  In SpEC, we called anything that computed something from the evolved fields an `Observer`, and a collection of test particles `Tracers`, and I don't see any reason to change that nomenclature.

Reply:
I agree with @kidder, we should stick to the current convention. You shouldn't be reading it in a GR context here anyway, it's more in a CS context :)